standards/bulk.md

```diff
@@ -0,0 +1,179 @@
+- `detail` field **MAY** be included as a complex object following the identified schema. If it is include, then it **MUST** include a `message` field. It **MUST NOT** be extended. The additional fields around `code`, `field`, and `value` are optional and **MAY** be left out if not applicable to your resource
+
+#### Example
+
+The following example demonstrates various operations in a bulk request and response:
+
+```json
+// REQUEST
+PATCH /articles
+Content-Type: application/json
+{
+    "operations": [
+        {
+            "action": "CREATE_UPDATE",
+            "ifMatch": "33a64df551425fcc55e4d42a148795d9f25f89d4",
+            "entity": {
+                "id": "bfd8f0c0-be67-4f81-bf82-e55e552609f4",
+                "name": "my name",
+                "description": "my description"
+            }
+        },
+        {
+            "action": "CREATE",
+            "operationId": "my-unique-id-or-uuid", 
+            "entity": {
+                "id": null,
+                "name": "my name",
+                "description": "my description"
+            }
+        },
+        {
+            "action": "DELETE",
+            "ifMatch": "44a64df551425fcc55e4d42a148795d9f25f89c5",     
+            "entity": {
+                "id": "d9bd5d91-fc25-4410-ae42-c8f631e8e9ff",
+                "name": null,
+                "description": null
+            }
+        }
+    ]
+}
+ 
+// RESPONSE
+200 OK
+Content-Type: application/json
+{
+    "status": "PARTIAL",                                 
+    "operations": [
+        {
+            "operationId": "0",
+            "action": "CREATE_UPDATE",
+            "entityId": "bfd8f0c0-be67-4f81-bf82-e55e552609f4",
+            "entityRef": "sps:thing:bfd8f0c0-be67-4f81-bf82-e55e552609f4",
+            "status": "SUCCEEDED",                         
+            "detail": {
+                "message": "Article was updated.",
+                "code": null,
+                "field": null,
+                "value": null
+            }
+        },
+        {
+            "operationId": "my-unique-id-or-uuid",
+            "action": "CREATE",
+            "entityId": null,
+            "entityRef": null,
+            "status": "FAILED",                         
+            "detail": {
+                "message": "Could not create article, since the name already exists.",
+                "code": "UNIQUE_NAME_VIOLATION",
+                "field": "name",
+                "value": "my name"
+            }  
+        },
+        {
+            "operationId": "2",
+            "action": "DELETE",
+            "entityId": "d9bd5d91-fc25-4410-ae42-c8f631e8e9ff",
+            "entityRef": "sps:thing:d9bd5d91-fc25-4410-ae42-c8f631e8e9ff",
+            "status": "SUCCEEDED",                         
+            "detail": null
+        }
+    ]
+}
```

Comment:
Looks like these examples are out of date now with the updated structure.

Reply:
wow :(  sorry Evan - been context switching way to much today and making sloppy mistakes. I appreciate your persistence in reviewing this for consistency!